command.go

```diff
@@ -102,6 +102,21 @@ type Command struct {
 	//   * PersistentPostRun()
 	// All functions get the same args, the arguments after the command name.
 	//
+	// When TraverseChildrenHooks is set, PersistentPreRun and
```

Comment:
Maybe worth adding something like `By default, if PersistentPreRun or PersistentPostRun is set, the parent's Pre/Post run hooks are not executed`?